code/script.js

```diff
@@ -30,18 +55,134 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+//Interaction 7, confirm all preferenses and suggest dog breeds.
+const dogBreed = (lifestyle, size, live, coat) => {
+  if(lifestyle === "couchPotato" && size === "small" && live === "citySuburb" && coat === "short" ) {
+  showMessage("Based on your preferences;<br>You're a Couch potato who live in a city or suburb and wants a small sized dog with a short coat.<br> You should get a short coated Chihuahua or a Pug!", "bot")
+  } else if(lifestyle === "couchPotato" && size === "small" && live === "citySuburb" && coat === "long" ) {
+    showMessage("Based on your preferences;<br>You're a Couch potato who live in a city or suburb and wants a small sized dog with a long coat.<br> You should get a long coated Chihuahua or a Pekingese!", "bot")
+  } else if(lifestyle === "couchPotato" && size === "small" && live === "countryside" && coat === "short" ) {
+    showMessage("Based on your preferences;<br>You're a Couch potato who live in a city or suburb and wants a small sized dog with a long coat.<br> You should get a short coated Chihuahua or a Pekingese!", "bot")
+  } else {
+  showMessage("I need to think about this a bit, come back later!", "bot")
+  }
+}
+
+//Interaction 6, confirm environment choice and ask what coat they prefer on a dog. Store value.
+const coatChoice = (live) => {
+  showMessage(`Ok, ${live} it is! Do you want a dog with a short or long coat?`, "bot")
+    inputWrapper.innerHTML = buttonSelectCoat
+    document
+      .getElementById("long")
+      .addEventListener("click", () => {
+      showMessage("I want a dog with a long coat", "user")
+      inputWrapper.innerHTML = ""
+      setTimeout(() => dogBreed("long"), 1000)
+  })
+  document
+      .getElementById("short")
+      .addEventListener("click", () => {
+      showMessage("I want a dog with a short coat", "user")
+      inputWrapper.innerHTML = ""
+      setTimeout(() => dogBreed("short"), 1000)
+  })
+  const coat = document.getElementById("submit")
+}
```

Comment:
**6.** And pick up three arguments here:
```const coatChoice = (lifestyleChoice, size, live) => {```
and pass along four arguments when you invoke the next function:
```setTimeout(() => dogBreed(lifestyleChoice, size, live, "short"), 1000)```